ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -786,6 +812,27 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
       })
     );
   },
+  removeSelectedOrgnizationItem: (
+    userMetadata: UserMetadata,
+    selectedOrganizations: ReadonlyArray<string>,
+    selectedOrganizationId: string
+  ) => {
+    const metadata = userMetadata.metadata;
+    const updatedAreasOfInterest = selectedOrganizations.filter(
+      item => item !== selectedOrganizationId
+    );
+    dispatch(
+      userMetadataUpsert.request({
+        ...userMetadata,
+        // tslint:disable-next-line: no-useless-cast
+        version: (userMetadata.version as number) + 1,
```

Comment:
Is this cast needed?

Reply:
I see that every time a user upsert is dispatched, the version is updated. Do you think it is useless in case of the removal of one item among the user areas of interest?

Reply:
Ops sorry, maybe my suggestion could be not so clear, I was referring to avoid cast

from this
```typescript
version: (userMetadata.version as number) + 1,
```

to this
```typescript
version: userMetadata.version + 1,
```

Reply:
I see it is required to pass the circleci `yarn tsc:noemit ` test. Updated within https://github.com/teamdigitale/io-app/pull/1330/commits/fe13f3a3c30f47627e2fc185bbe125a26680edee as:

```
const item: number = useMetadata.version;
...
version: item + 1
```